config/initializers/carrierwave.rb

```diff
@@ -5,6 +5,6 @@
       aws_access_key_id: ENV["AWS_ACCESS_KEY_ID"],
       aws_secret_access_key: ENV["AWS_SECRET_ACCESS_KEY"]
     }
-    config.fog_directory  = ENV["S3_BUCKET"]
+  config.fog_directory  = ENV["S3_BUCKET"]
```

Comment:
I actually meant to move the `=` closer to the `config.fog_directory`.

Basically, `config.fog_directory  = ENV["S3_BUCKET"]` -> config.fog_directory = ENV["S3_BUCKET"]`. I didn't explain myself very well, so no harm done.